example/40-validatingwebhookconfiguration.yaml

```diff
@@ -17,7 +17,9 @@ webhooks:
     - shoots
     - cloudprofiles
   failurePolicy: Fail
-  objectSelector: {}
+  objectSelector:
```

Comment:
Here it is fine to rely on it, I guess. Maybe you can add a comment that at least Gardener v1.42 is needed.